web/src/components/pages/dashboard/dashboard.tsx

```diff
@@ -169,10 +186,28 @@ function DashboardContent({
   return <Page {...{ allGoals, dashboardLocale }} />;
 }
 
+const ChallengeBar = () => (
+  <div className="challenge-bar">
+    <div className="points">
+      <img src={require('./awards/star.svg')} alt="score" />
+      <span className="score">448</span>
+      <span className="label label-my"></span>
+      <span className="divider"></span>
+      <span className="score">12345</span>
+      <span className="label label-team"></span>
+    </div>
+    <Button rounded className="inviteButton">
+      <span className="content">Invite</span>
+      <span className="plusIcon"></span>
+    </Button>
+  </div>
+);
+
 const PAGES = [
   { subPath: URLS.STATS, Page: StatsPage },
   { subPath: URLS.GOALS, Page: GoalsPage },
   { subPath: URLS.AWARDS, Page: AwardsPage },
+  { subPath: URLS.CHALLENGE, Page: AwardsPage },
```

Comment:
Can we try rendering `StatsPage` here instead? `AwardsPage` has some visual bugs (it's full-width, unlike the other pages), so I think `StatsPage` will give a better idea of what the layout will actually look like.

Thanks!